ui/src/routes/devices.$id.mount.tsx

```diff
@@ -374,8 +374,8 @@ function UrlView({
       icon: FedoraIcon,
     },
     {
-      name: "openSUSE Leap 15.6",
-      url: "https://download.opensuse.org/distribution/leap/15.6/iso/openSUSE-Leap-15.6-NET-x86_64-Media.iso",
+      name: "openSUSE Leap 16.0",
+      url: "https://download.opensuse.org/distribution/leap/16.0/offline/Leap-16.0-online-installer-x86_64.install.iso",
```

Comment:
Shouldn't that be `https://download.opensuse.org/distribution/leap/16.0/offline/Leap-16.0-offline-installer-x86_64.install.iso` (note the 16.0-offline vs 16.0-online)?

Also, I'm getting a 403 for all releases at https://get.opensuse.org/leap/16.0 so this release might not actually be valid?